crates/uv-cache/src/lib.rs

```diff
@@ -535,6 +535,9 @@ pub enum CacheBucket {
     /// TODO(konstin): The cache policy should be on the source distribution level, the metadata we
     /// can put next to the wheels as in the `Wheels` bucket.
     ///
+    /// The unzipped source distribution is stored in a directory matching the source distribution
+    /// acrhive name.
+    ///
     /// Source distributions are built into zipped wheel files (as PEP 517 specifies) and unzipped
```

Comment:
There is more in this docstring that should change but I do not feel like I have the context on the changes to the cache design to do so.